src/status_im/subs/wallet/buy.cljs

```diff
@@ -5,3 +5,15 @@
  :wallet/crypto-on-ramps
  :<- [:wallet]
  :-> :crypto-on-ramps)
+
+(rf/reg-sub
+ :wallet/one-time-crypto-on-ramps
+ :<- [:wallet/crypto-on-ramps]
+ (fn [crypto-on-ramps _]
+   (filter #(seq (:site-url %)) crypto-on-ramps)))
+
+(rf/reg-sub
+ :wallet/recurrent-crypto-on-ramps
+ :<- [:wallet/crypto-on-ramps]
+ (fn [crypto-on-ramps _]
+   (filter #(seq (:recurrent-site-url %)) crypto-on-ramps)))
```

Comment:
Still related, and I don't how much the feedback is useful, but we could store separate pieces of crypto on ramp results (recurrent or on time for example) in the app-db and do the filtering/removal once, at write time (event layer), instead of read time (sub layer). It's just an idea, not a suggestion for change.

Reply:
Really appreciate the suggestions. I will update the code. Thank you @ilmotta!